### ISSUE TITLE
schema:title, schema:name and jobTitle

### ISSUE THREAD

**Issue by marco-brandizi** [schemaorg/schemaorg#2983]
[schema:title](https://schema.org/title) is defined as: "The title of the job", however, 'title' in the URI is a much more generic term and usually, it refers to whatever title an entity might have (book title, film title, `dcterms:title`).  

Moreover, [schema:jobTitle](https://schema.org/title) exists too and [schema:name](https://schema.org/name) seems to be intended as one of name, label (it's defined as subproperty of `rdfs:label`) or title (it's defined as equivalent to `dct:title`). This isn't very good, since often a name is a short name for an entity and not necessarily the same as a title or an ID, eg, the [protein with name: TP53](https://en.wikipedia.org/wiki/P53) might also have the title: 'tumor protein p53' and the ID/accession: ENSG00000141510.  

I would propose to redefine `schema:title` adopting a generic definition, and to separate `schema:name` from dct:title (possibly, from rdfs:label as well).

**Comment by RichardWallis** [comment]
Taking the properties you reference individually:

- [schema:name](https://schema.org/name)  is the generic 'name' property for all types - Thing, Place, Volcano, Vehicle, Book, etc.  It is defined (see the [More..] option) as being _Equivalent Property: dct:title_ which helps when relating to artefacts that have titles such as books, but many more things have names than titles such as people and volcanos.
- [schema:title](https://schema.org/title) "_The title of the job._" is only for use on the [JobPosting](https://schema.org/JobPosting) type.  A bit specific usage of a generic property name.
- [schema:jobTitle](https://schema.org/jobTitle) "_The job title of the person (for example, Financial Manager)._" is only for use on the [Person](https://schema.org/Person) type.

There is some historical baggage here as to how properties have been named.  Taking account of the description and domainIncludes of each helps clarify things somewhat.

**Comment by marco-brandizi** [comment]
@RichardWallis, thanks for your comment. Yes, the domain clarifies that `schema:title` is for a job posting, but in my opinion, we hardly want something as generic as "title" to be constrained into such a specific case, it would be worth to generalise it and then it could be used for `JobPosting` too (or `jobTitle` could be used for that).

**Comment by danbri** [comment]
We should update https://schema.org/title to point to the other properties
like /name that carry titles-of-pages and similar.

At this stage I don't think we'll help matters by introducing a generic
/title which means same thing as /name

**Comment by marco-brandizi** [comment]
I can see the current modelling, yet I don't know what to say. I continuously meet examples where I have to distinguish between a short name and a title (see above) and I'd like to have two different properties for that (possibly with a common parent like `rdfs:label`).

I also understand the will to ensure backward compatibility, but that's not always possible and here the impact on existing datasets should be minimal.

**Comment by dr-shorthair** [comment]
I agree that 'shortName' is a common requirement. Particularly for drop-down menus. It would be nice to have a generic property.
